### ISSUE TITLE
A bit confused

### ISSUE THREAD

**Issue by smileBeda** [iamkevinrhodes/goldbootstrap#1]
The only thing that I can do with this is use golden h's and cold-gold-lg and gold-sm?

nothing else? like col-gold-md, row, container, btn-gold... erc etc 

The menu, buttons, everything looks the same to me as in BT3.0 if I use your bootstrap.css/js files

am I wrong, nuts or just overseeing something?

How would I code a Theme with your BT?
I mean semantics like container, article, main, masthead, etc etc... is all the same as in BT, right?

So, basically the only that changes with Gold BT is the h's and the 2 col's you mention in your demo-site?

Thank you, I like the idea of golden stuff, but I think we could go far further?

**Comment by kevinrhodesdesign** [comment]
The line-height property was changed to 1.618, which slightly changes padding and spaces on virtually every single element of Bootstrap.

The headings and columns were the only 2 pure "additions".

**Comment by smileBeda** [comment]
I see.

Nice, at least I am not stupid then :dancer: 

about the col's, i see it right that only gold-lg and gold-sm can be used, right?

so, I mean, -12, -9 etc, or col-sm make no sense with "gold", right?

Thanks :)

**Comment by kevinrhodesdesign** [comment]
Exactly. Only 2 Columns make up the "golden ratio". Take a look the column graphic on gold bootstrap.com to understand what I mean.
